packages/utils/telemetry-utils/src/config.ts

```diff
@@ -166,6 +166,24 @@ const safeSessionStorage = (): Storage | undefined => {
 	}
 };
 
+/**
+ * Creates a wrapper on top of an existing config provider which allows for
+ * specifying feature gates if not present in the original provider.
+ *
+ * @param defaults - default feature gate configs to be used if not specified by the original provider
+ * @param original - the original config provider
+ * @returns The value for the requested feature gate from the original provider and if not present,
+ * from the specified feature gates
+ *
+ * @alpha
```

Comment:
This can be @internal to begin with, right? Our partners can't be using it now and I'd say we don't want to expose it as part of our API.

Reply:
that's the feedback I was looking for. Thanks!